File: Web/Divyanshu-Vareshney_divyanshuvarshney_2125it1075_2/GettingStarted/Reverse A Number.cpp
#include <iostream>

using namespace std;

int main(int argc, char **argv){
    int n;
    cin >> n;

    //write your code here
    int i;
    while(n>0){
        i=n%10;
        cout<<i<<"\n";
        n=n/10;
    }
    //donee
    
}

File: Android/Garima0812S_GarimaShukla_2125ec1114_2/GettingStarted/ReverseANumber.java
import java.util.*;
   
   public class Main{
   
   public static void main(String[] args) {
     // write your code here  
      Scanner sc=new Scanner(System.in);
          int n=sc.nextInt();
          int c=0;
          while(n>0)
          {
              c=n%10;
              System.out.println(c);
              n=n/10;
          }

    }
   }